.changeset/new-gorillas-film.md

```diff
@@ -0,0 +1,5 @@
+---
+'@firebase/app-check': patch
+---
+
+Update App Check to v1 from v1beta for both reCAPTCHA v3 and reCAPTCHA Enterprise
```

Comment:
Can you change to "Update App Check to use v1 endpoint instead of v1beta endpoint..."? So it doesn't sound like we're updating the version of the App Check SDK itself.

Reply:
Yes, I just updated the wording